pkgs/top-level/all-packages.nix

```diff
@@ -9943,7 +9943,16 @@ let
 
   ekho = callPackage ../applications/audio/ekho { };
 
-  electrum = callPackage ../applications/misc/electrum { };
+  electrum = callPackage ../applications/misc/electrum {
```

Comment:
This is quite ugly. What about accepting pythonPackages in the electrum expression.